src/tracing/trace_event_common.h

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#ifndef TRACE_EVENT_COMMON_H
```

Comment:
Personally I'd prefer `#pragma once`, but I yield to the authors preference.

If we're staying with `#ifndef` IMO it should be `SRC_TRACING_TRACE_EVENT_COMMON_H`, similar to other file in this directory:
https://github.com/nodejs/node/blob/2d5387e143255d97210b25280df1addb7f5ccb34/src/tracing/node_trace_buffer.h#L1
as per:
https://github.com/nodejs/node/blob/2d5387e143255d97210b25280df1addb7f5ccb34/tools/cpplint.py#L1959-L1965

Reply:
Personally I think `#pragma once` is more clean and tidy.

But most libs `#ifndef`, it just looks more beautiful to me :)